arcgis-ios-sdk-samples/Maps/Browse building floors/BrowseBuildingFloorsViewController.swift

```diff
@@ -0,0 +1,115 @@
+// Copyright 2021 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class BrowseBuildingFloorsViewController: UIViewController {
+    // MARK: Storyboard views
+    
+    /// The map view managed by the view controller.
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            mapView.map = makeMap()
+        }
+    }
+    /// The floor level picker view.
+    @IBOutlet var floorLevelPickerView: UIPickerView!
+    
+    // MARK: Properties
+    
+    /// The levels of the floor-aware web map, with an addition of `nil`.
+    var levels: [AGSFloorLevel?] = [nil]
+    
+    // MARK: Methods
+    
+    /// Set the selected level to visible and all other levels to invisible.
+    /// - Parameter selectedLevel: The `AGSFloorLevel` selected.
+    func setVisibleFloorLevel(_ selectedLevel: AGSFloorLevel?) {
+        levels.forEach { level in
+            level?.isVisible = level == selectedLevel
+        }
+    }
+    
+    func setPickerViewLayout() {
+        floorLevelPickerView.translatesAutoresizingMaskIntoConstraints = false
+        let margin: CGFloat = 10.0
+        floorLevelPickerView.layer.cornerRadius = margin
+        NSLayoutConstraint.activate([
+            floorLevelPickerView.widthAnchor.constraint(equalToConstant: 120.0),
+            floorLevelPickerView.bottomAnchor.constraint(equalTo: mapView.attributionTopAnchor, constant: -margin),
+            floorLevelPickerView.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor, constant: -margin)
```

Comment:
Consider using system spacing.
```suggestion
            view.safeAreaLayoutGuide.trailingAnchor.constraint(equalToSystemSpacingAfter: floorLevelPickerView.trailingAnchor, multiplier: 1),
            mapView.attributionTopAnchor.constraint(equalToSystemSpacingBelow: floorLevelPickerView.bottomAnchor, multiplier: 1)
```

Reply:
Thanks for pointing this out! New method learned... 😸